src/viam/robot/client.py

```diff
@@ -81,19 +81,20 @@ class RobotClient:
         from viam.robot.client import RobotClient
 
 
-        async def connect():
-            opts = RobotClient.Options.with_api_key(
-                # Replace "<API-KEY>" (including brackets) with your machine's API key
-                api_key='<API-KEY>',
-                # Replace "<API-KEY-ID>" (including brackets) with your machine's API key ID
-                api_key_id='<API-KEY-ID>'
-            )
-            return await RobotClient.at_address('<ADDRESS-FROM-THE-VIAM-APP>', opts)
-
+        async def connect(address) -> RobotClient:
+            opts = RobotClient.Options(
+                disable_sessions=True, dial_options=DialOptions(timeout=10)).with_api_key(
+                    # Replace "<API-KEY>" (including brackets) with your API key
+                    api_key='<API-KEY>',
+                    # Replace "<API-KEY-ID>" (including brackets) with your API key
+                    # ID
+                    api_key_id='<API-KEY-ID>'
+                )
+            return await RobotClient.at_address(address=address, options=opts)
 
         async def main():
             # Make a RobotClient
-            machine = await connect()
+            machine = await connect('ADDRESS FROM CODE SAMPLE TAB OF VIAM APP')
```

Comment:
I did this in a code sample I sent you because I needed to connect to many machines and so was passing that in as an argument, that's not needed here
```suggestion
            machine = await connect()
```